pype/tools/settings/__init__.py

```diff
@@ -1,4 +1,4 @@
-from settings import style, MainWidget
+from .settings import style, MainWidget
```

Comment:
I think it is not allowed to do relative import here if launched from `__main__`.

Reply:
sure, but settings and tray are now launched like modules like `python -m pype.tools.settings`